### PR TITLE
Fix some security issues from dependencies

### PR DESCRIPTION
**Pull request by msmol**
I don't think we need a version bump of the SDK for this -- the next time we have a real update it will include these changes.

### CODE REVIEW

**Review by cdifulvio — APPROVED** [review]

**Review by shwetasabne — APPROVED** [review]

File: build/sdk/js/templates/package.mustache [hunk 1]
```diff
@@ -20,5 +20,8 @@
     "mocha": "^6.1.4",
     "sinon": "1.17.3",
     "uuid": "^3.3.2"
+  },
+  "resolutions": {
+    "*/**/js-yaml": "^3.13.1"
```
Comment (anthonyjonahs) [inline build/sdk/js/templates/package.mustache:25]: what is this for?

**Review by anthonyjonahs — APPROVED** [review]

File: build/sdk/js/templates/package.mustache (hunk 1 above)
Comment (msmol) [reply build/sdk/js/templates/package.mustache:25]: do you mean what is "resolutions" for or what is "js-yaml" for?

Comment (anthonyjonahs) [reply build/sdk/js/templates/package.mustache:25]: `resolutions`

Comment (msmol) [reply build/sdk/js/templates/package.mustache:25]: discussed offline -- merging this

**Pull request merged**